src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditingPreferencesTests.java

```diff
@@ -27,6 +25,8 @@
   private static final String SOURCE = "1";
   private static final String VE = "2";
   private static final String CK = "3";
```

Comment:
me neither :( what's more interesting, these tests are never run on jenkins. After a quick investigation I found out they should be owned by Sus, filed https://wikia-inc.atlassian.net/browse/QAART-959 for that - let them handle the refactoring where needed